sysutils/diffutils/Portfile

```diff
@@ -51,20 +50,26 @@ use_xz              yes
 #   die (EXIT_TROUBLE, 0, "%s", _("write failed"));
 #   ^
 #------------------------------------------------------------------------------
-compiler.blacklist-append \
-                    { clang < 900 }
+compiler.blacklist-append   { clang < 900 }
 
-depends_build-append \
-                    port:gettext
+depends_build-append        port:gettext
 
-depends_lib-append \
-                    port:gettext-runtime
+depends_lib-append          port:gettext-runtime
 
-patchfiles          patch-src_system.h.diff
-configure.args      --program-prefix=g \
-                    --infodir=${prefix}/share/info \
-                    --mandir=${prefix}/share/man
+patchfiles                  patch-src_system.h.diff
 
+# Bypass year 2038 awareness code that causes the universal variant to fail.
+pre-configure {
+                            if [variant_isset "universal"] {
```

Comment:
Remember to [brace your expressions](https://wiki.tcl-lang.org/page/Brace+your+expr%2Dessions).

Reply:
Thanks.  I am still learning tcl.